// The Firmament project
// Copyright (c) 2011-2012 Malte Schwarzkopf <malte.schwarzkopf@cl.cam.ac.uk>
//
// Task-related messages, convering task creation and task completion.

package firmament;

import "messages/base_message.proto";

extend BaseMessage {
  optional TaskHeartbeatMessage task_heartbeat_extn = 104;
}

message TaskHeartbeatMessage {
  required uint64 task_id = 1;
  optional string location = 2;
  optional uint64 sequence_number = 3;
}
